libethcore/Precompiled.cpp

```diff
@@ -132,15 +132,34 @@ ETH_REGISTER_PRECOMPILED(modexp)(bytesConstRef _in)
 	return {true, ret};
 }
 
+namespace
+{
+	bigint expLengthAdjust(bigint _baseLength, bigint _expLength, bytesConstRef _in)
+	{
+		size_t const numConsideredBytes(min(_expLength, {32}));
```

Comment:
Also, better pass `_expOffset` into this function instead of `_baseLength` and leave `expOffset = baseLength + 96` calculation in the calling function